### PR TITLE
Add variables to overwrite the spark resource configs

### PR DESCRIPTION
**Pull request by NvTimLiu**
Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
Local test PASS

File: examples/micro-benchmarks/notebooks/micro-benchmarks-gpu.ipynb [hunk 1]
```diff
@@ -92,7 +97,7 @@
     "# Pinned 8g host memory to transfer data between GPU and host memory\n",
     "conf.set(\"spark.rapids.memory.pinnedPool.size\", \"8G\")\n",
```
[the remark this answers is not in the document]
Comment (NvTimLiu) [reply examples/micro-benchmarks/notebooks/micro-benchmarks-gpu.ipynb]: fixed

File: examples/micro-benchmarks/notebooks/micro-benchmarks-cpu.ipynb [hunk 2]
```diff
@@ -503,9 +506,9 @@
...
     " \n",
-    "conf.set(\"spark.driver.memory\", \"50g\")\n",
-    "conf.set(\"spark.executor.memory\", \"6g\")  \n",
-    "conf.set(\"spark.executor.cores\", \"2\") \n",
+    "conf.set(\"spark.driver.memory\", driverMem)\n",
+    "conf.set(\"spark.executor.memory\", executorMem)\n",
+    "conf.set(\"spark.executor.cores\", executorCores)\n",
     " \n",
     "conf.set(\"spark.locality.wait\", \"0\")\n",
     "conf.set(\"spark.sql.files.maxPartitionBytes\", \"2g\") \n",
```
[the remark this answers is not in the document]
Comment (NvTimLiu) [reply examples/micro-benchmarks/notebooks/micro-benchmarks-cpu.ipynb]: fixed

File: examples/micro-benchmarks/notebooks/micro-benchmarks-cpu.ipynb [hunk 3]
```diff
@@ -63,16 +63,19 @@
...
     "conf.setAppName(\"Microbenchmark on CPU\")\n",
-    "conf.set(\"spark.driver.memory\", \"50g\")\n",
-    "conf.set(\"spark.executor.memory\", \"100g\")  \n",
-    "conf.set(\"spark.executor.cores\", \"32\") \n",
+    "conf.set(\"spark.driver.memory\", driverMem)\n",
+    "conf.set(\"spark.executor.memory\", executorMem)\n",
+    "conf.set(\"spark.executor.cores\", executorCores)\n",
     " \n",
     "conf.set(\"spark.locality.wait\", \"0\")\n",
     "conf.set(\"spark.sql.files.maxPartitionBytes\", \"1g\") \n",
```
[the remark this answers is not in the document]
Comment (NvTimLiu) [reply examples/micro-benchmarks/notebooks/micro-benchmarks-cpu.ipynb]: fixed

**Review by nvliyuan — APPROVED** [review]

**Comment by NvTimLiu** [comment]
Local test PASS

**Pull request merged**